.ci/azure/pipelines/obs-packaging-ci.yml

```diff
@@ -11,9 +11,48 @@ pool:
   vmImage: 'ubuntu-latest'
 
 steps:
-- script: echo setup
-  displayName: 'setup CI'
+- task: DockerInstaller@0
```

Comment:
OOI, what is the `@0` bit?

```
^ ^
0@0
\-/
```

Reply:
Azure magic for installing docker I think, as per the link now added to the yml? https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/docker-installer?view=azure-devops